spring/spring-data/src/test/java/io/quarkus/ts/spring/data/di/SpringDiTest.java

```diff
@@ -21,6 +22,7 @@
 // TODO: Workaround by implementing REST endpoints which expose all necessary information.
 @Tag("fips-incompatible") // native-mode
 @QuarkusTest
+@Disabled
```

Comment:
Add the reason, please

Reply:
Sorry, this was a testing relic. Reverted.